app/features/sendouq-settings/routes/q.settings.tsx

```diff
@@ -659,3 +662,46 @@ function SoundCheckboxes() {
     </div>
   );
 }
+
+function SoundSlider() {
+  const [volume, setVolume] = useState(() => {
+    return soundVolume();
+  });
+
+  useEffect(() => {
+    setVolume(soundVolume());
+  }, []);
```

Comment:
This combined with `defaultValue` does not look quite correct 🤔 I tried removing useEffect and I see no change in functionality. But then again updating state is pointless if you are using defaultValue afaik https://react.dev/reference/react-dom/components/input#controlling-an-input-with-a-state-variable

I would suggest:

- use null (or 100 doesn't matter) as default value in useState
- `defaultValue` -> `value`

Reply:
I implemented useEffect because of the isBrowser check initially, but youre right that it is no longer needed!
Will refactor to use value instead